tools/parser/helpers.go

```diff
@@ -132,3 +135,62 @@ func ParseMsigParams(msg *filTypes.Message, height int64, key filTypes.TipSetKey
 
 	return parsedParams, nil
 }
+
+func (p *Parser) parseAccount(txType string, msg *filTypes.Message, msgRct *filTypes.MessageReceipt) (map[string]interface{}, error) {
+	switch txType {
+	case "Send":
+		return p.parseSend(msg), nil
+	case "PubkeyAddress":
+		metadata := make(map[string]interface{})
+		metadata[tools.ParamsKey] = base64.StdEncoding.EncodeToString(msg.Params)
+		return metadata, nil
+	}
+	return map[string]interface{}{}, errUnknownMethod
+}
+
+func (p *Parser) parseSend(msg *filTypes.Message) map[string]interface{} {
+	metadata := make(map[string]interface{})
+	metadata[tools.ParamsKey] = msg.Params
+	return metadata
+}
+
+//func (p *Parser) parseExec(msg *filTypes.Message, msgRct *filTypes.MessageReceipt,
```

Comment:
Do you need an example to test this one? or why is commented out ?

Reply:
oh, my bad there. That function has been replaced for this one https://github.com/Zondax/filecoin-indexing-rosetta-proxy/pull/32/files#diff-b70287623d9bc6cf64324eefa7111cb2bd90498b1b6160c2cbcbea95a1d28110R37-R62